app/models/CreditDetailModel.scala

```diff
@@ -16,19 +16,25 @@
 
 package models
 
-import models.financialDetails.{BalanceDetails, CodingDetails, DocumentDetail, FinancialDetail, FinancialDetailsModel}
+import models.financialDetails.{DocumentDetail, FinancialDetailsModel}
 import models.paymentAllocationCharges.FinancialDetailsWithDocumentDetailsModel
```

Comment:
I would think that using compiler option can a good option; but this could lead to confusion in the future for someone to support this code, I might think to drop implicit usage totally to call things directly